static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -0,0 +1,105 @@
+import {ProductSolution} from 'sentry/components/onboarding/productSelection';
+
+type ProductSelectionMap = Record<ProductSolution, boolean>;
+
+/**
+ * Transforms the product selection array into a map of booleans for each product for easier access.
+ */
+export const getProductSelectionMap = (
+  activeProductSelection: ProductSolution[]
+): ProductSelectionMap => {
+  const productSelectionMap: ProductSelectionMap = {
+    [ProductSolution.ERROR_MONITORING]: false,
+    [ProductSolution.PROFILING]: false,
+    [ProductSolution.PERFORMANCE_MONITORING]: false,
+    [ProductSolution.SESSION_REPLAY]: false,
+  };
+
+  activeProductSelection.forEach(product => {
+    productSelectionMap[product] = true;
+  });
+  return productSelectionMap;
+};
+
+/**
+ * Joins the given lines with the given indentation using \n as delimiter.
+ */
+export function joinWithIndentation(lines: string[], indent = 2) {
+  const indentation = ' '.repeat(indent);
+  return lines.map(line => `${indentation}${line}`).join('\n');
+}
+
+export function getInstallSnippet({
+  productSelection,
+  basePackage = '@sentry/node',
+}: {
+  productSelection: ProductSelectionMap;
+  basePackage?: string;
+}) {
+  const packages = [basePackage];
+  if (productSelection.profiling) {
+    packages.push('@sentry/profiling-node');
+  }
+  return `# Using yarn
+yarn add ${packages.join(' ')}
+
+# Using npm
+npm install --save ${packages.join(' ')}`;
+}
+
+export function getDefaultNodeImports({
+  productSelection,
+}: {
+  productSelection: ProductSelectionMap;
+  basePackage?: string;
+}) {
+  const imports: string[] = [
+    `import * as Sentry from '@sentry/node'`,
+    `// You can also use CommonJS \`require('@sentry/node')\` instead of \`import\``,
```

Comment:
l: Wdyt about switching the order here? Looking at the screenshot i feel like it's a little awkward that the comment is in between imports. If this doesn't play well with the dynamic code snippet generation, feel free to leave it as is.
 
```suggestion
    `// You can also use CommonJS \`require('@sentry/node')\` instead of \`import\``,
    `import * as Sentry from '@sentry/node'`,    
```